src/auth.js

```diff
@@ -142,22 +142,41 @@ export default class DropboxAuth {
   }
 
   generatePKCECodes() {
-    let codeVerifier = crypto.randomBytes(PKCELength);
-    codeVerifier = codeVerifier.toString('base64')
-      .replace(/\+/g, '-')
-      .replace(/\//g, '_')
-      .replace(/=/g, '')
-      .substr(0, 128);
+    let codeVerifier;
+    function convertToStringAndReplace(toBeConverted) {
+      const convertedString = toBeConverted.toString('base64')
+        .replace(/\+/g, '-')
+        .replace(/\//g, '_')
+        .replace(/=/g, '');
+      return convertedString;
+    }
+
+    if (isBrowserEnv()) {
+      const array = new Uint32Array(10);
+      crypto.getRandomValue(array)
+        .then((randomValueArray) => {
+          codeVerifier = convertToStringAndReplace(randomValueArray).substr(0, 128);
```

Comment:
The array won't convert directly to a string as far as I am aware, try to print it both from browser and from node and ensure that you get the same format of strings